### ISSUE TITLE
does sanitizer have been implimented in flowdroid

### ISSUE THREAD

**Issue by mengjianwei12345** [secure-software-engineering/FlowDroid#516]
I found that there are test cases for sanitizer in the securibench of the flowdroid, so I wanted to ask if the flowdroid analysis can already support the sanitizer for harmless treatment

**Comment by flankerhqd** [comment]
I guess you can implement this using summary or TaintPropagationHandler.

**Comment by StevenArzt** [comment]
Correct. The StubDroid summaries already support "kill flows". have a look at the summary for ``List.clear()`` (forlder ``soot-infoflow-summaries/summariesManual``).
